### PR TITLE
Simplify settings names

### PR DESCRIPTION
**Pull request by janfaracik**
Follow up from #7314. This PR simplifies the names of settings in Manage Jenkins by dropping unnecessary prefixes, such as 'Manage' and 'Configure'. This makes it easier to scan the page to find what your looking for, and also makes the search functionality results clearer.

image

### Testing done
* Browsed Manage Jenkins ensuring the new names are there

### Proposed changelog entries

- Simplify settings names in Manage Jenkins.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7661"></a>

### CODE REVIEW

File: core/src/main/resources/jenkins/management/Messages.properties
Comment (timja) [inline core/src/main/resources/jenkins/management/Messages.properties:25]: Please remove all translations that are no longer appropriate or re-translate them, e.g. https://github.com/janfaracik/jenkins/blob/d52812a8f9e86465195f3d0d8f1ba6056f01f27a/core/src/main/resources/jenkins/management/Messages_bg.properties#L23 and many others

**Review by timja — COMMENTED** [review]
Screenshot looks good, translations need adjusting or removing.

Can you send a pull request to the credentials plugin to update it to match the conventions and review this list to see if there's any other PRs that need to be made:
https://github.com/janfaracik/jenkins/blob/d52812a8f9e86465195f3d0d8f1ba6056f01f27a/core/src/main/resources/jenkins/management/Messages_bg.properties#L23

----

Also ATH will need running to make sure nothing was clicking via link text and is now broken

**Comment by janfaracik** [comment]
> Can you send a pull request to the credentials plugin to update it to match the conventions 

Opened a PR here - https://github.com/jenkinsci/jenkins/pull/7661#pullrequestreview-1304851044

Thanks

**Comment by daniel-beck** [comment]
Please file a corresponding PR to jenkins.io that updates documentation accordingly.

**Review by uhafner — APPROVED** [review]
The new screenshot looks good! Thanks for cleaning up the names.

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Review by timja — APPROVED** [review]
ATH Passed

----

/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
